server/cli/flags_test.go

```diff
@@ -35,3 +36,31 @@ func TestStdFlagToPflag(t *testing.T) {
 		}
 	})
 }
+
+func TestNoLinkTesting(t *testing.T) {
+	imports := make(map[string]struct{})
+
+	var addImports func(string)
+	addImports = func(root string) {
```

Comment:
Doesn't work because recursive!
On May 6, 2015 3:54 PM, "Tobias Schottdorf" <notifications@github.com>
wrote:

> In server/cli/flags_test.go
> <https://github.com/cockroachdb/cockroach/pull/937#discussion_r29795183>:
>
> > @@ -35,3 +36,31 @@ func TestStdFlagToPflag(t *testing.T) {
> >  		}
> >  	})
> >  }
> > +
> > +func TestNoLinkTesting(t *testing.T) {
> > +	imports := make(map[string]struct{})
> > +
> > +	var addImports func(string)
> > +	addImports = func(root string) {
>
> nitnit: addImports := .... would probably be preferable, but just leave
> it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/937/files#r29795183>.
>